File: Proyectos  c++ (2)/Descuento.cpp
#include<iostream>
#include<locale.h>

using namespace std;

int main(){
setlocale(LC_CTYPE, "spanish");
		int ca , vu , m , p , p2 , ds , ds2 ;
		cout<< "Vamos conocer cuanto debe pagar un cliente por la compra de una serie de productos" <<endl;
		cout<< "Ingrese la cantidad de articulos: " <<endl;
		cin>> ca;
		cout<< "Ingrese el valor unitario: " <<endl;
		cin>> vu;
		m = ca * vu;
		p = m * 15/100;
		p2 = m * 35/100;
		ds = m - p;
		ds2 = m - p2;
		if (m>20000)
		{
			cout<<"El total a pagar es: "<<ds2 ;
			cout<<"\nYa que se le aplica el 35% de descuento ";
		}
		else
		{
			cout<<"El total a pagar por los productos es: "<<ds ;
			cout<<"\nYa que se le aplica el 15% de descuento ";
		}
}

File: Proyectos Python (2)/Descuento.py
print ("Programa que va a determinar el total a pagar por una compra")
cp = int( input ("Ingrese la cantidad de articulos: "))
vu = int ( input ("Ingrese el valor unitario de los productos: "))

if cp*vu>=20000:
    print("El total a a pagar por la compra de los productos es: " ,(cp*vu-cp*vu*35/100)," Ya que se le descuenta el 35% de descuento que es: " ,(cp*vu*35/100))
else:
    print("El total a pagar por la compra de productos es: ",(cp*vu-cp*vu*15/100)," Ya que se le descuenta el 10% de descuento que es: " ,(cp*vu*35/100))
